src/dotnet-core-uninstall/Windows/ListCommandExec.cs

```diff
@@ -28,12 +20,38 @@ public static GridView GetGridView(IList<Bundle> bundles)
             foreach (var (bundle, index) in bundles.Select((bundle, index) => (bundle, index)))
             {
                 gridView.SetChild(new ContentView(string.Empty), 0, index);
-                gridView.SetChild(new ContentView(bundle.Version.ToString()), 1, index);
+                gridView.SetChild(new ContentView(bundle.Version.ToString(true)), 1, index);
                 gridView.SetChild(new ContentView(bundle.Arch.ToString().ToLower()), 2, index);
                 gridView.SetChild(new ContentView($"\"{bundle.DisplayName}\""), 3, index);
             }
 
             return gridView;
-        }
+        };
+
+        private static readonly Func<IList<Bundle>, GridView> _gridViewGeneratorWithoutArch = bundles =>
```

Comment:
nit: I don't have a better idea yet. However,  Func<IList<Bundle>, GridView>,ListCommandExec, and ListListCommandExecs are not easy to follow. And I don't have a good way to reason about their responsibility individually